ci/test/00_setup_env.sh

```diff
@@ -71,3 +71,7 @@ export CI_BASE_PACKAGES=${CI_BASE_PACKAGES:-build-essential libtool autotools-de
 export GOAL=${GOAL:-install}
 export DIR_QA_ASSETS=${DIR_QA_ASSETS:-${BASE_SCRATCH_DIR}/qa-assets}
 export CI_RETRY_EXE=${CI_RETRY_EXE:-"retry --"}
+if [ "$CI_OS_NAME" == "macos" ]; then
```

Comment:
This file is currently only generic exports. Can we avoid adding platform specific conditionals?

Reply:
It seems it might be moved into `./ci/test/04_install.sh`. What do you think?

Reply:
Reworked.